x-pack/plugin/stack/src/main/java/org/elasticsearch/xpack/stack/StackTemplateRegistry.java

```diff
@@ -53,6 +53,36 @@
         TEMPLATE_VERSION_VARIABLE
     );
 
+    //////////////////////////////////////////////////////////
+    // Built in ILM policies for users to use
+    //////////////////////////////////////////////////////////
+    public static final String ILM_7_DAYS_POLICY_NAME = "7-days-hot-warm";
+    public static final String ILM_30_DAYS_POLICY_NAME = "30-days-hot-warm";
+    public static final String ILM_90_DAYS_POLICY_NAME = "90-days-hot-warm-cold";
+    public static final String ILM_180_DAYS_POLICY_NAME = "180-days-hot-warm-cold";
+    public static final String ILM_365_DAYS_POLICY_NAME = "365-days-hot-warm-cold";
```

Comment:
We might want to avoid spelling out which phases we are using in the name of the policy, so that we could later change this, e.g. if we later realize that it would make sense to use `frozen` when users keep data for 365 days?

Reply:
I think it'd be nice to avoid those names, but currently we don't have a concept of a "title" of a policy from within ILM or the UI, so if we made those less general, it would be much harder for a user to determine what the policy does (especially since these are intended for users not that familiar with ILM) if we named it even more generic like "365-days".

Do you think the lack of specification for upgrades is better, or would it be better to be more specific so a user can tell what it is doing?

Reply:
I'd rather like to be less specific so that we can more easily change our mind on what we think are good defaults. Maybe the name could be something like `180-days-default` instead of `180-days` to make it clearer that there are multiple ways to retain data for 180 days and that this is just one way of doing it?

Reply:
Sounds good, I've renamed the policies.